app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationInfo.jsx

```diff
@@ -111,7 +111,9 @@ export const IntegrationInfo = (props) => {
         id: 'createProjectIntegrationModal',
         data: {
           modalTitle: formatMessage(messages.projectIntegrationCreate),
-          hasGlobalIntegration: Boolean(availableGlobalIntegrations.length),
+          hasGlobalIntegration: Boolean(
+            availableGlobalIntegrations.length && availableProjectIntegrations.length === 0,
+          ),
```

Comment:
Right now `hasGlobalIntegration` is not clear name as you depends on not only on the availableGlobalIntegrations

Reply:
I think hasGlobalIntegration -> hasWarningMessage will be better, what do you think?